[2023-06-26T12:01:36.804+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:01:32.482465+00:00/task_id=load_seed_todl_once permission to 509
[2023-06-26T12:01:36.806+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:01:32.482465+00:00/task_id=load_seed_todl_once permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:01:32.482465+00:00/task_id=load_seed_todl_once'
[2023-06-26T12:01:36.840+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:01:32.482465+00:00/task_id=load_seed_todl_once permission to 509
[2023-06-26T12:01:36.841+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:01:32.482465+00:00/task_id=load_seed_todl_once permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:01:32.482465+00:00/task_id=load_seed_todl_once'
[2023-06-26T12:01:36.903+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5.load_seed_todl_once manual__2023-06-26T12:01:32.482465+00:00 [queued]>
[2023-06-26T12:01:36.922+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5.load_seed_todl_once manual__2023-06-26T12:01:32.482465+00:00 [queued]>
[2023-06-26T12:01:36.923+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-26T12:01:36.952+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): load_seed_todl_once> on 2023-06-26 12:01:32.482465+00:00
[2023-06-26T12:01:36.960+0000] {standard_task_runner.py:57} INFO - Started process 74 to run task
[2023-06-26T12:01:36.966+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5', 'load_seed_todl_once', 'manual__2023-06-26T12:01:32.482465+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpbn3kywi3']
[2023-06-26T12:01:36.970+0000] {standard_task_runner.py:85} INFO - Job 115: Subtask load_seed_todl_once
[2023-06-26T12:01:37.058+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:01:32.482465+00:00/task_id=load_seed_todl_once permission to 509
[2023-06-26T12:01:37.059+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:01:32.482465+00:00/task_id=load_seed_todl_once permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5/run_id=manual__2023-06-26T12:01:32.482465+00:00/task_id=load_seed_todl_once'
[2023-06-26T12:01:37.063+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5.load_seed_todl_once manual__2023-06-26T12:01:32.482465+00:00 [running]> on host 84990cf1ddee
[2023-06-26T12:01:37.248+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5' AIRFLOW_CTX_TASK_ID='load_seed_todl_once' AIRFLOW_CTX_EXECUTION_DATE='2023-06-26T12:01:32.482465+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-26T12:01:32.482465+00:00'
[2023-06-26T12:01:37.251+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-26T12:01:37.252+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt &&             dbt seed --profiles-dir .']
[2023-06-26T12:01:37.266+0000] {subprocess.py:86} INFO - Output:
[2023-06-26T12:01:41.653+0000] {subprocess.py:93} INFO - [0m12:01:41  Running with dbt=1.5.1
[2023-06-26T12:01:51.156+0000] {subprocess.py:93} INFO - [0m12:01:51  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-26T12:01:51.157+0000] {subprocess.py:93} INFO - There are 2 unused configuration paths:
[2023-06-26T12:01:51.158+0000] {subprocess.py:93} INFO - - models.badge_5.create_stage
[2023-06-26T12:01:51.159+0000] {subprocess.py:93} INFO - - models.badge_5.time_of_day
[2023-06-26T12:01:51.222+0000] {subprocess.py:93} INFO - [0m12:01:51  Found 1 model, 0 tests, 0 snapshots, 0 analyses, 455 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-26T12:01:51.230+0000] {subprocess.py:93} INFO - [0m12:01:51
[2023-06-26T12:01:53.771+0000] {subprocess.py:93} INFO - [0m12:01:53  Concurrency: 200 threads (target='dev')
[2023-06-26T12:01:53.772+0000] {subprocess.py:93} INFO - [0m12:01:53
[2023-06-26T12:01:53.852+0000] {subprocess.py:93} INFO - [0m12:01:53  1 of 1 START seed file ***.time_of_day_lu .................................. [RUN]
[2023-06-26T12:01:56.224+0000] {subprocess.py:93} INFO - [0m12:01:56  1 of 1 OK loaded seed file ***.time_of_day_lu .............................. [[32mINSERT 24[0m in 2.37s]
[2023-06-26T12:01:56.318+0000] {subprocess.py:93} INFO - [0m12:01:56
[2023-06-26T12:01:56.319+0000] {subprocess.py:93} INFO - [0m12:01:56  Finished running 1 seed in 0 hours 0 minutes and 5.08 seconds (5.08s).
[2023-06-26T12:01:56.481+0000] {subprocess.py:93} INFO - [0m12:01:56
[2023-06-26T12:01:56.483+0000] {subprocess.py:93} INFO - [0m12:01:56  [32mCompleted successfully[0m
[2023-06-26T12:01:56.485+0000] {subprocess.py:93} INFO - [0m12:01:56
[2023-06-26T12:01:56.488+0000] {subprocess.py:93} INFO - [0m12:01:56  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2023-06-26T12:01:57.246+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-26T12:01:57.283+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5, task_id=load_seed_todl_once, execution_date=20230626T120132, start_date=20230626T120136, end_date=20230626T120157
[2023-06-26T12:01:57.373+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-26T12:01:57.421+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
